src/di.php

```diff
@@ -744,7 +753,7 @@
     }
 
     $cookieBBlang = $di['cookie']->get('BBLANG');
-    $locale = !empty($cookieBBlang) ? $cookieBBlang : $di['config']['locale'];
+    $locale = !empty($cookieBBlang) ? $cookieBBlang : $di['config']['i18n']['locale'];
```

Comment:
Needs a fallback if it's undefined. This will do:
`$locale = !empty($cookieBBlang) ? $cookieBBlang : ($di['config']['i18n']['locale'] ?? 'en_US');`

Reply:
Added 🙂 .